src/commands/get_command.py

```diff
@@ -0,0 +1,26 @@
+import json
+from commands.abstract_command import *
+from output.exceptions import *
+from model.database import *
+
+
+class GetCommand(AbstractCommand):
+
+    def __init__(self, database, table, query):
+        GetCommand.validate(database, table)
+        self.database = database
+        self.table = table
+        if query is None or query == "" or query == " ":
```

Comment:
(Minor comment ) Better to use this 
`test_str1 and not test_str1.isspace())): `
Also it is better make it in one line